src/main/windows/mainWindow.ts

```diff
@@ -74,7 +84,7 @@ function createMainWindow(options: {linuxAppIcon: string}) {
         minWidth: MINIMUM_WINDOW_WIDTH,
         minHeight: MINIMUM_WINDOW_HEIGHT,
         frame: !isFramelessWindow(),
-        fullscreen: savedWindowState.fullscreen,
+        fullscreen: isFullScreen(options.fullscreen ?? savedWindowState.fullscreen),
```

Comment:
```suggestion
        fullscreen: isFullScreen(),
```
if we apply the changes above, we don't need any parameters here.